Assets/Scripts/GameMan.cs

```diff
@@ -200,6 +207,42 @@ public void RpcCardMoved(GameObject card, CardAreas destination)
         card.transform.rotation = Quaternion.Euler(0, 0, 0);
     }
 
+    [ClientRpc]
+    public void RpcUpdateClientHand(GameObject card)
+    {
+        trickColor = card.GetComponent<Card>().GetColor();
+        Debug.Log("trick color set to " + trickColor);
+        bool hasLeadColor = false;
+        // playerArea.GetComponent<Image>().color = new Color32(255,255,225,100);
+        for (int i = 0; i < playerArea.transform.childCount; i++)
+        {
+            Card handCard = playerArea.transform.GetChild(i).gameObject.GetComponent(typeof(Card)) as Card;
+            if (handCard.GetColorAlways() == trickColor)
+            {
+                hasLeadColor = true;
+                continue;
+            }
+        }
+        if (hasLeadColor)
+        {
+            Debug.Log("has trick color");
+            for (int i = 0; i < playerArea.transform.childCount; i++)
+            {
+                Card handCard = playerArea.transform.GetChild(i).gameObject.GetComponent(typeof(Card)) as Card;
+                handCard.SetTrickColor(trickColor);
+            }
+
+        } else
+        {
+            Debug.Log("all cards playable");
+            for (int i = 0; i < playerArea.transform.childCount; i++)
+            {
+                Card handCard = playerArea.transform.GetChild(i).gameObject.GetComponent(typeof(Card)) as Card;
+                handCard.SetPlayable(true);
+            }
+        }
+    }
+
```

Comment:
This section will probably have a simpler solution after the cards list have been implemented in one of the other branches; since it works, I think this is good!